config/settings.yml

```diff
@@ -284,14 +284,22 @@ modules_appeals_api:
       api_key: ''
   # The token generation values below are only used for development rake tasks.
   token_generation:
```

Comment:
These settings values are only used in local development workflows, so there's nothing related to update in the deployed environments